### PR TITLE
add npc that sells all imbuiment items

### PR DESCRIPTION
**Pull request by czbadaro**
adds a NPC that sells imbuiment items

### CODE REVIEW

**Review by dudantas — APPROVED** [review]

**Comment by MajestyOTBR** [comment]
Fix npc name and remove `script="testserver_assistant.lua"` because there is no lua script.

**Comment by MajestyOTBR** [comment]
Fix npc name and set `script="testserver_assistant.lua"` to `script=default.lua"`.

**Comment by majestyotbr** [comment]
Fix npc name and set `script="testserver_assistant.lua"` to `script="default.lua"`.

File: data/npc/imbuiment_assistent.xml [hunk 1]
```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<npc name="Imbuiment Assistant" walkinterval="2000" floorchange="0" script="testserver_assistant.lua">
```
Comment (dudantas) [inline data/npc/imbuiment_assistent.xml]:
```suggestion
<npc name="Imbuement Assistant" walkinterval="2000" floorchange="0" script="default.lua">
```

**Review by dudantas — CHANGES_REQUESTED** [review]

**Comment by majestyotbr** [comment]
PR closed due to inactivity.

**Pull request closed without merging**